scenes/quests/story_quests/stealth_template/stealth_template.tscn

```diff
@@ -0,0 +1,129 @@
+tile_set = ExtResource("2_apejx")
+
+[node name="Player" parent="." instance=ExtResource("3_5de16")]
+position = Vector2(200, 432)
+
+[node name="Camera2D" type="Camera2D" parent="Player"]
+unique_name_in_owner = true
+process_mode = 3
+zoom = Vector2(1.7, 1.7)
+limit_left = 0
+limit_top = 0
+position_smoothing_enabled = true
+editor_draw_limits = true
+
+[node name="StealthQuest" type="Node2D" parent="."]
+y_sort_enabled = true
+script = ExtResource("4_u1h50")
+
+[node name="EnemyGuards" type="Node2D" parent="StealthQuest"]
+unique_name_in_owner = true
+y_sort_enabled = true
+
+[node name="Guard-GoingBackAndForth" parent="StealthQuest/EnemyGuards" node_paths=PackedStringArray("patrol_path") instance=ExtResource("5_x7wv1")]
+position = Vector2(334, 497)
+patrol_path = NodePath("../Guard-PatrolPath")
+move_speed = 200.0
+
+[node name="Guard-PatrolPath" type="Path2D" parent="StealthQuest/EnemyGuards"]
+top_level = true
+position = Vector2(1022, 221)
+scale = Vector2(0.107143, 0.970109)
+curve = SubResource("Curve2D_vnsq3")
+
+[node name="Guard6-RunningInCircles" parent="StealthQuest/EnemyGuards" node_paths=PackedStringArray("patrol_path") instance=ExtResource("5_x7wv1")]
+modulate = Color(1, 0, 0, 1)
+position = Vector2(1999, 242)
+patrol_path = NodePath("../Guard6-PatrolPath")
+wait_time = 0.3
+move_speed = 1000.0
+detection_area_scale = 0.1
+
+[node name="Guard6-PatrolPath" type="Path2D" parent="StealthQuest/EnemyGuards"]
+position = Vector2(-1737, -21.8142)
+scale = Vector2(1, 1.08777)
+curve = SubResource("Curve2D_46hcm")
+
+[node name="Checkpoints" type="Node2D" parent="."]
+
+[node name="Checkpoint" parent="Checkpoints" instance=ExtResource("6_366c6")]
+position = Vector2(1357, 300)
+
+[node name="CollisionShape2D" type="CollisionShape2D" parent="Checkpoints/Checkpoint"]
+position = Vector2(396, 138)
+shape = SubResource("RectangleShape2D_idy4y")
+debug_color = Color(0, 0, 0, 0.42)
+
+[node name="Transition" type="CanvasLayer" parent="."]
+
+[node name="TransitionFade" type="ColorRect" parent="Transition"]
+unique_name_in_owner = true
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+color = Color(0, 0, 0, 0)
+metadata/_edit_lock_ = true
+
+[node name="ScreenOverlay" type="CanvasLayer" parent="."]
+
+[node name="HUD" parent="." instance=ExtResource("7_bqwsu")]
+
+[node name="CollectibleItem" parent="." instance=ExtResource("8_wu3if")]
+position = Vector2(2335, 955)
+scene_to_go_to = "uid://cd2p2udmkif80"
```

Comment:
@jbourqueendless This is the wrong target scene. I suggest that you either link it to nothing (until all 3 templates are landed and we can string the 5 scenes together), or to the outro.